src/Utility.ts

```diff
@@ -115,6 +116,25 @@ export namespace Utility {
         return restartConfig && (httpPort !== newHttpPort || httpsPort !== newHttpsPort);
     }
 
+    export async function readFileLineByLine(file: string, filterFunction?: (value: string) => boolean): Promise<string[]> {
+        let result: string[] = [];
```

Comment:
let -> const

Reply:
result will be concated with each line ```result = result.concat(line);``` while reading the file